check_tv.py

```
@@ -61,7 +61,7 @@ def offline_check(driver, live_url, spare_link, important):
       fewtimes = 0
       if config.Twitch == "True":
         try:
-          driver.find_element("xpath", "//button[@data-a-target='content-classification-gate-overlay-start-watching-button']//div[text()='開始觀看']").click()
+          driver.find_element("xpath", "//button[@data-a-target='content-classification-gate-overlay-start-watching-button']").click()
           time.sleep(5)
         except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)